gef.py

```diff
@@ -6313,7 +6313,8 @@ def do_invoke(self, argv):
             if get_libc_version() < (2, 30):
                 count = ord(read_memory(addr + i, 1))
             else:
-                count = ord(read_memory(addr + 2 * i, 1))
+                # Type of tcache_perthread_struct.counts is (uint16_t *), not (char *). Just changing the offset is not enough.
+                count = struct.unpack('{}H'.format(endian_str()), read_memory(addr + 2 * i, 2))[0]
```

Comment:
Good stuff.

However, some helpers were added to make such syntax a bit lighter so please rebase your GEF branch after commit aac9021921b759965cce2b5f8ab381629757d5b0 and change the line to

```python
                count = u16( read_memory(addr + 2 * i, 2) )
```

instead.

Thanks!